lib/dictionary.js

```diff
@@ -10,9 +10,9 @@ var segmenter = new Segmenter(definitionLookup);
 
 function start(){
 	console.log("Hanzi is compiling dictionary...");
-	
+
 	//Reading in CCEDICT
-	var readFile = fs.readFileSync(__dirname + '/dicts/cedict_ts.u8', 'utf-8');
```

Comment:
Do you have a spaces vs tabs preference? Some files in this repo use both

Reply:
see #35 that address the indenting issue.

The tabs in the frequency files have not been changed in this PR

Reply:
Awesome @zurawiki, would you recommend we merge in PR #35 first and then update this PR to match the new indenting? What do you think?